examples/dev/external_provisioner/remote-local-exec_swarm-cluster/main.tf

```diff
@@ -0,0 +1,115 @@
+terraform {
+  required_providers {
+    grid = {
+      source = "threefoldtech/grid"
+    }
+  }
+}
+
+provider "grid" {
+}
+
+locals {
+  name = "myvm"
+}
+resource "grid_capacity_reserver" "cap1" {
+  farm   = 1
+  cpu    = 4
+  memory = 2048
+  ssd    = 50
+
+}
+resource "grid_network" "net1" {
+  nodes         = [1]
```

Comment:
this field is computed and should be replaced with `capacity_ids` with capacity id from `cap1`